common/content/statusline.js

```diff
@@ -130,8 +130,7 @@ const StatusLine = Module("statusline", {
         if ( typeof this.setVisibility.currVisibility == 'undefined' ) {
             this.setVisibility.currVisibility = true;
             this.setVisibility.prevVisibility = true;
-            this.setVisibility.contentSeparator
-                = highlight.get('ContentSeparator').value;
+            this.setVisibility.passiveVisibility = false;
```

Comment:
Whether or not we are in fullscreen, currVisibility can have any boolean
value. This variable cannot be inferred from anything else. But
passiveVisibility is possible to be inferred from whether we are in command
mode.

2016년 2월 2일 (화) 14:13, Gary Katsevman <notifications@github.com>님이 작성:

> In common/content/statusline.js
> <https://github.com/vimperator/vimperator-labs/pull/407#discussion_r51525040>
> :
>
> > @@ -130,8 +130,7 @@ const StatusLine = Module("statusline", {
> >          if ( typeof this.setVisibility.currVisibility == 'undefined' ) {
> >              this.setVisibility.currVisibility = true;
> >              this.setVisibility.prevVisibility = true;
> > -            this.setVisibility.contentSeparator
> > -                = highlight.get('ContentSeparator').value;
> > +            this.setVisibility.passiveVisibility = false;
>
> Ah, I see. I wonder if there's a way to get whether we're in fullscreen
> and use that for checking against rather than adding another moving part.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vimperator/vimperator-labs/pull/407/files#r51525040>.
>